all-in-one/.swcrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Is this meant to be in the examples?

Reply:
No I just opened a PR to fix this in the template.

Reply:
PR https://github.com/functionless/create-functionless/pull/8